# DRINKO 

Drinko is a company that operates on the fast-moving consumer goods industry **FMCG**


The FMCG industry stands for Fast-Moving Consumer Goods industry, which is a large and diverse sector that deals with the production, distribution, and marketing of frequently consumed consumer products such as food, beverages, personal care, household products, and other such products.

FMCG products are typically low-cost, high-volume products that are in constant demand and have a short shelf life. These products are sold through multiple distribution channels, including supermarkets, convenience stores, and online platforms. FMCG companies typically invest heavily in branding, advertising, and product promotion to drive consumer demand and maintain their market share.

The FMCG industry is highly competitive, with many large multinational companies, such as Procter & Gamble, Nestle, Unilever, and Coca-Cola, competing for market share. The industry is also highly regulated, with companies required to meet stringent quality standards and adhere to various regulations regarding packaging, labeling, and product safety.

## Important questions

 our dashboard will have to answer:-

**Which are their most profitable products?**


**What drives their customers to pick
one product or another?**


**What is the optimal size of the item
that clients prefer to buy?**

### **The dataset **

https://github.com/RaedOkal/DRINKO_Case_Study/blob/main/excel_dashboard_raw_data%20(1).xlsx





**The spreadsheet you see on your screen represents a data extraction that is typically obtained when working with a so-called ERP system**

**ERP stands for enterprise resource planning these are integrated software solutions that a company uses to collect store and manage data recorded as a result of business activity therefore Erp systems are data warehouses something like a container that collects the entire information of a firm** 

**the two most popular ERP system providers in the world are Oracle and sap**

## let's examine the data that we have

In [2]:
import pandas as pd 

In [3]:
df=pd.read_excel('excel_dashboard_raw_data (1).xlsx')

In [4]:
df.head()

Month  Year  Material Number  ... Cost of Goods Sold  Distribution Warehousing
0      1  2020         11504000  ...         -68.761481     -4.034226   -1.479495
1      1  2020         11506000  ...         -40.603423     -2.271397   -0.833003
2      1  2020         11508000  ...         -59.650237     -1.965568   -0.720844
3      1  2020         11660150  ...         -25.011177     -1.367352   -0.501457
4      1  2020         11660180  ...         -37.888523     -1.446658   -0.530541

[5 rows x 17 columns]

In [5]:
print(df.dtypes)

Month                     int64
Year                      int64
Material Number           int64
Material Description     object
Period                    int64
Brand                    object
Size                     object
Pack                     object
Client                   object
Client Type              object
Volume                  float64
Gross Sales             float64
Discounts               float64
Net Sales               float64
Cost of Goods Sold      float64
Distribution            float64
Warehousing             float64
dtype: object


**The extraction you see here has been made in Sap's business objects application business objects is a tool that allows you to extract data that has been stored in the system** 

**The data is naturally downloaded and used in Excel by the firms financial analyst as you can see every row of the extraction contains specific descriptive and financial information**


In [6]:
descriptive_Fields = df.iloc[:5, :5]
descriptive_Fields

Month  Year  Material Number Material Description  Period
0      1  2020         11504000      Crocky 250ML 4X  202001
1      1  2020         11506000      Crocky 250ML 6X  202001
2      1  2020         11508000      Crocky 250ML 8X  202001
3      1  2020         11660150     Crocky 330ML 15X  202001
4      1  2020         11660180     Crocky 330ML 18X  202001

**In the left part of the sheet are the so-called descriptive Fields showing descriptive information like the material number description the period when this occurred**

In [7]:
df.iloc[:5, 5:10]

Brand   Size Pack  Client   Client Type
0  Crocky  250ML   4X  Kroger  Supermarkets
1  Crocky  250ML   6X  Kroger  Supermarkets
2  Crocky  250ML   8X  Kroger  Supermarkets
3  Crocky  2.25L  15X  Kroger  Supermarkets
4  Crocky  2.25L  18X  Kroger  Supermarkets

**then we have the relevant brand size pack client and client type for the specific row**

In [9]:
df.iloc[:5, 10:]

Volume  Gross Sales  ...  Distribution  Warehousing
0  206.000000   170.802637  ...     -4.034226    -1.479495
1  147.865268    96.167296  ...     -2.271397    -0.833003
2  104.924074    83.219005  ...     -1.965568    -0.720844
3   72.990660    57.891482  ...     -1.367352    -0.501457
4   77.224118    61.249188  ...     -1.446658    -0.530541

[5 rows x 7 columns]

**on the right side is all the financial information we will need in this exercise**

It starts with **volume : the amount that was sold per each of these lines**

In [21]:
df[['Month', 'Brand' ,'Size','Pack', 'Client', 'Client Type', 'Volume']].head()


Month   Brand   Size Pack  Client   Client Type      Volume
0      1  Crocky  250ML   4X  Kroger  Supermarkets  206.000000
1      1  Crocky  250ML   6X  Kroger  Supermarkets  147.865268
2      1  Crocky  250ML   8X  Kroger  Supermarkets  104.924074
3      1  Crocky  2.25L  15X  Kroger  Supermarkets   72.990660
4      1  Crocky  2.25L  18X  Kroger  Supermarkets   77.224118

in our first example this shows that 206 units of the **Brand** crucky with **size** 250 ml four pack we're sold in Kroger supermarkets in January

In [22]:
df[[ 'Client', 'Client Type', 'Volume','Gross Sales','Discounts','Net Sales']].head()

Client   Client Type      Volume  Gross Sales  Discounts   Net Sales
0  Kroger  Supermarkets  206.000000   170.802637  -3.416053  167.710929
1  Kroger  Supermarkets  147.865268    96.167296  -0.961673   94.426566
2  Kroger  Supermarkets  104.924074    83.219005  -1.664380   81.712653
3  Kroger  Supermarkets   72.990660    57.891482  -1.157830   56.843585
4  Kroger  Supermarkets   77.224118    61.249188  -1.837476   60.140513

**We can see the various Financial measures** 

**Gross sales show a firms hypothetical revenues without any discounts** 

**If we then consider the discounts that had been given to clients we will arrive at net sales**


 **It is a common practice that clients receive different discounts based on the amount of business they generate this is a really important item because the size of discounts has a direct impact on the price that a client (a supermarket) for example can offer to its customers  This practice is also known as volume-based pricing, where the more a client purchases, the greater discount they receive**

In [23]:
df[['Volume','Gross Sales','Discounts','Net Sales','Cost of Goods Sold','Distribution','Warehousing']].head()

Volume  Gross Sales  ...  Distribution  Warehousing
0  206.000000   170.802637  ...     -4.034226    -1.479495
1  147.865268    96.167296  ...     -2.271397    -0.833003
2  104.924074    83.219005  ...     -1.965568    -0.720844
3   72.990660    57.891482  ...     -1.367352    -0.501457
4   77.224118    61.249188  ...     -1.446658    -0.530541

[5 rows x 7 columns]

**On the right side of the table we have three additional cost items cost of goods sold distribution and warehousing expenses**

**Cost of goods sold (COGS) shows how much was spent to produce** 

**The product distribution and warehousing are two items that are often grouped together** 

**Distribution is the transport expenditure that has to be sustained to deliver a product to a client**

**Warehousing indicates the cost of keeping the item in the company's storage facilities until the item is delivered to clients**

### Side Notes about the dashboard

# mapping with find and replace

**the final product **

https://github.com/RaedOkal/DRINKO_Case_Study/blob/main/excel_dashboard_raw_data%20(1).xlsx